### PR TITLE
Darken labels and fix col gap under profile

### LINKED ISSUE #2566: Make text labels under profile darker
Make text labels in the profile darker

Currently, background and foreground colours do not have a sufficient contrast ratio.

image

### PR DESCRIPTION
**Pull request by rabilrbl**
Fixes #2566

| Before | After |
| - | - |
| image | image |

- less code repetitions

### CODE REVIEW

**Comment by nihal467** [comment]
@rabilrbl @gigincg the deploy preview failed

File: src/Components/Users/UserProfile.tsx [hunk 1]
```diff
@@ -301,6 +301,29 @@ export default function UserProfile() {
       setTimeout(() => setUpdateBtnText("Update"), 1000);
     }
   };
+
+  const LabelValueCard = (props: { children: React.ReactNode }) => (
```
Comment (developedBySJ) [inline src/Components/Users/UserProfile.tsx]: Avoid Declaring React Component Inside Parent Component.
 
 Move component outside

**Review by developedBySJ — CHANGES_REQUESTED** [review]

**Comment by rithviknishad** [comment]
@rabilrbl deploy preview is failing. merge the latest develop branch.

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Comment by khavinshankar** [comment]
@rabilrbl please do work on the merge conflicts

**Review by developedBySJ — APPROVED** [review]
You can keep components within a file if they are closely related or only used once.

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**